samples/Common.c

```diff
@@ -1116,7 +1116,7 @@ STATUS signalingMessageReceived(UINT64 customData, PReceivedSignalingMessage pRe
     }
 
 CleanUp:
-
+	SAFE_MEMFREE(pReceivedSignalingMessageCopy);
```

Comment:
This will cause an access violation as you are freeing a piece of memory that you've added to the queue to be processed later. The fix should be somewhat different.

Reply:
https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/blob/27b95a928c05810f901949fa02364acdb17b16e3/samples/Common.c#L1096-L1102

In fact, it ignores the else condition, the _pPendingMessageQueue_ will be nill, so the _pReceivedSignalingMessageCopy_ will not be put into the queue.

Reply:
Will be submitting a fix asap